src/main/java/seedu/address/model/company/Date.java

```diff
@@ -47,7 +49,7 @@ public static boolean isValidDate(String test) {
 
     @Override
     public String toString() {
-        return date.toString();
+        return this.date.toString();
```

Comment:
Do you think it will be better to not use "this" since we have not used it elsewhere?

Reply:
done